pkg/api/universal/state_query.go

```diff
@@ -16,8 +16,14 @@ package universal
 import (
 	"context"
 	"encoding/json"
+	httpCodes "net/http"
```

Comment:
Please don't rename `http` to `httpCodes` as this package includes more than the codes, and if someone needs something else from `net/http` in the future, this will be confusing

Reply:
there were 2 http imports from what I saw earlier, but that may have just been a local issue. I'll switch this back 👍🏻